### PR TITLE
LRQA 70033

### PR DESCRIPTION
**Pull request by guivilarim**
- LRQA-70033 Create test stubs
- LRQA-70033 Create assertWebContentNotPresent, assertDataInWebContentField and inputDataInWebContentField macros
- LRQA-70033 Add CanInputLink test
- LRQA-70033 Add CanClearInput test
- LRQA-70033 Add CanNotPublishBlankRequiredField test
- LRQA-70033 Add CanRemoveDuplicatedField test
- LRQA-70033 Add CanInputDataOnDuplicatedField test
- LRQA-70033 Add test descriptions

### CODE REVIEW

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEWebContentField.testcase [hunk 1]
```diff
@@ -81,12 +81,41 @@ definition {
...
 
+		DERenderer.inputDataInWebContentField(
+			fieldLabel = "Web Content",
+			webContentWebContent = "Web Content Title 1");
+
+		Button.clickPublish();
+
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
+
+		DERenderer.clearImageField(fieldLabel = "Web Content");
```
Comment (natocesarrego) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEWebContentField.testcase:110]: @guivilarim, I think it's time to rename this macro, after all you're cleaning a Web Content Field and not an Image Field.

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEWebContentField.testcase [hunk 2]
```diff
@@ -203,7 +203,7 @@ definition {
 			webContentWebContent = "Web Content Title 2");
 	}
 
-	@description = "This is a stubbed description"
+	@description = "This is a test for LRQA-70033. This test verifies that is possible to link an Web Content"
```
Comment (natocesarrego) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/fields/DEWebContentField.testcase:206]: @guivilarim, what exactly do you mean with **link an Web Content**? Can you please clarify?

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Comment by natocesarrego** [comment]
Good job, @guivilarim! :slightly_smiling_face:

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
Hey @guivilarim, can you please fix the source formatter issues and resend? Thanks

**Pull request closed without merging**